beancount/utils/misc_utils.py

```diff
@@ -281,15 +282,16 @@ def compute_unique_clean_ids(strings):
     return idmap
 
 
-def escape_string(string):
-    """Escape quotes and backslashes in payee and narration.
+def escape_string(string: str) -> str:
+    """Escape string in double quoted style, the returned str itself not quoted.
+    for example: `escape_string("1\n\"2") -> "1\\n\"2"`
```

Comment:
This is not the style in which docstrings need to be written: the first line needs to be a self standing paragraph. Anyhow "double quoted style" is not a term with a recognizable meaning.
```suggestion
    """Escape backslashes, double quotes, and control characters."""
```